idlpy/src/context.c

```diff
@@ -569,7 +570,7 @@ static idl_retcode_t write_module_headers(FILE *fh, idlpy_ctx octx, idlpy_module
     }
 
     idl_fprintf(fh, "]\n");
-    idlpy_ssos_free(modules);
+    free(modules);
```

Comment:
Line 501 (new numbering) allocates it with `idlpy_ssos_new`, so it is really surprising that it should be freed with `free` instead of `idlpy_ssos_free`.